src/Markdig/Extensions/NoRefLinks/NoRefLinksExtension.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Alexandre Mutel. All rights reserved.
+// This file is licensed under the BSD-Clause 2 license. 
+// See the license.txt file in the project root for more information.
+
+using Markdig.Renderers;
+using Markdig.Renderers.Html.Inlines;
+
+namespace Markdig.Extensions.NoRefLinks
+{
+    public class NoRefLinksExtension : IMarkdownExtension
```

Comment:
Could you add a comment to this?

Also could you add the associated `UseNotRefLinks()` method extension to [MarkdownExtension.cs](https://github.com/lunet-io/markdig/blob/master/src/Markdig/MarkdownExtensions.cs) (don't forget also the `Configure` method that should accept a string like `noreflinks`)

Sorry to bump into these details, once I will have written the doc, that should be easier. 

Thanks again, this is the first community extension! :+1: